test/e2e/specs/metrics_server.go

```diff
@@ -29,13 +29,7 @@ var _ = Describe("Openshift on Azure metrics server tests [MetricsServer][EveryP
 
 	It("should should not return for a pod [CustomerAdmin][MetricsServer]", func() {
 		By("retrieving metrics for pod master-api-master-000000")
-		_, err := sanity.Checker.Client.CustomerAdmin.MetricsServerV1beta1.PodMetricses("kube-system").Get("master-api-master-000000", metav1.GetOptions{})
-		Expect(err.Error()).To(Equal("pods.metrics.k8s.io \"master-api-master-000000\" is forbidden: User \"customer-cluster-admin\" cannot get pods.metrics.k8s.io in the namespace \"kube-system\": no RBAC policy matched"))
-	})
-
-	It("should should return valid metrics a given pod when accessed as a cluster admin [MetricsServer]", func() {
```

Comment:
I think this is the right test name and this needs to be updated?

Reply:
yes, the name is definitely wrong, thanks for catching it